benchmarks/BenchmarkServer/BenchmarkServer.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Hub\**" />
+    <Content Remove="Hub\**" />
+    <EmbeddedResource Remove="Hub\**" />
+    <None Remove="Hub\**" />
```

Comment:
![https://media.giphy.com/media/zjQrmdlR9ZCM/giphy.gif](https://media.giphy.com/media/zjQrmdlR9ZCM/giphy.gif)

Reply:
Please to explain? Or just remove?

Reply:
Haha, I renamed the folder from Hub to Hubs, wtf VS!